app/src/main/res/values/strings.xml

```diff
@@ -606,7 +609,8 @@
     <string name="view_public_repositories_button">View community repositories</string>
     <string name="view_public_repositories_button_short">Public list</string>
     <string name="uppercase_all_subtitles">Uppercase all subtitles</string>
-    <string name="download_all_plugins_from_repo">Download all plugins from this repository?</string>
+    <string name="download_all_plugins_from_repo">%1$s has been added. You can install your desired extensions from 𝗦𝗲𝘁𝘁𝗶𝗻𝗴𝘀 > 𝗘𝘅𝘁𝗲𝗻𝘀𝗶𝗼𝗻𝘀 > %1$s.
```

Comment:
The key for this NEEDS to be changed, as all languages other than english will just be confusing because they have not translated this yet. Also this is a bit long and no user will read this entire shit and will just press OK. Make it easier to read and remove `CloudStream 3 does not takes any responsibility for using third-party extensions neither provides support for them`